### ISSUE TITLE
Changing sphinx theme din't recreate index.html

### ISSUE THREAD

**Issue by jahan01** [rtfd/readthedocs.org#1980]
i change my theme from alabaster to sphinx_rtd_theme and rebuild my project.

The theme changed got effective in rest of the pages except for my index.html
I tried wiping out and rebuilding it dint work out either.

Link: http://dj-extensions.readthedocs.org/en/latest/index.html

Is that something intended or an bug?

Regards

**Comment by agjohnson** [comment]
Cached maybe? I get a page with our theme.

**Comment by jahan01** [comment]
This issue went once docs got rebuilt when I updated the docs content.

On this particular issue I just changed only html theme in conf.py (dint update any contents in rst files) and  theme was not updated only for index.html. Weirdly I was able to see updated theme in http://dj-extensions.readthedocs.org/en/latest/ which point to index but not in http://dj-extensions.readthedocs.org/en/latest/index.html

I think it was not caching issue on client side, because I tried multiple browsers (also in private mode) and 2 machines. Even I tried wiping and rebuilding the docs the issue persisted until I pushed a new change to docs rst files.

I dint grab a screenshot to show you now, though.
